test/stage-options.test.ts

```diff
@@ -2,6 +2,7 @@ import { readFileSync } from 'fs';
 import { Stack, Stage } from 'aws-cdk-lib';
 import { ShellStep } from 'aws-cdk-lib/pipelines';
 import { GitHubWorkflow, StackCapabilities } from '../src';
+import { GithubActionStep } from '../src/steps/GithubActionStep';
```

Comment:
We need to expose this in `index.ts` and then I think we can just import from `../src`. Otherwise, `GitHubActionStep` won't actually be exported.

Reply:
updated